### PR TITLE
Modified Readme

### PR DESCRIPTION
**Pull request by KrishAryan**
Signed-off-by: Krish Aryan <64402456+KrishAryan@users.noreply.github.com>

## Changes proposed
Improves the looks and functionality of Kubesimplify readme

Added light/dark mode logos 
Added badges
Added  contribution info

## Screenshots
image

image
image

## Note to reviewers
I made this new pull request because the old pull request #127 had some issues in merging.

### CODE REVIEW

**Comment by KrishAryan** [comment]
@AvineshTripathi please review this

**Comment by AvineshTripathi** [comment]
sure doing the review

File: README.md [hunk 1]
```diff
@@ -1,12 +1,39 @@
...
+  <source media="(prefers-color-scheme: light)" srcset="https://raw.githubusercontent.com/cncf/artwork/master/other/cncf/horizontal/black/cncf-black.svg" width="30%" align="center" style="margin-bottom:20px;">
+  <img alt="Shows an illustrated light mode kubesimplify logo in light color mode and a dark mode kubesimplify logo dark color mode." src="ttps://raw.githubusercontent.com/cncf/artwork/master/other/cncf/horizontal/white/cncf-white.svg" width="30%" align="center" style="margin-bottom:20px;">
+</picture></a><br /></p>
+<p align="center">
+Participation in the Kubesimplify community is governed by the [CNCF Code of Conduct](https://github.com/cncf/foundation/blob/master/code-of-conduct.md).
 
 <h1>Contributing Guidelines: </h1>
 
+We're a warm and welcoming community of open source contributors. Please join. All types of contributions are welcome. Be sure to read the Kubesimplify <a href="https://github.com/KrishAryan/website/blob/main/CONTRIBUTION.md">Contribution.md</a> for a tour of resources available to you and how to get started.
```
Comment (AvineshTripathi) [inline README.md:36]: please change the href ref

**Comment by KrishAryan** [comment]
@AvineshTripathi changed the href

File: README.md [hunk 2]
```diff
@@ -1,12 +1,39 @@
...
+    <a href="https://www.youtube.com/c/saiyam911" alt="Youtube">
+  <img src="https://img.shields.io/badge/YouTube-FF0000?style=for-the-badge&logo=youtube&logoColor=white" /></a>
+  </p>
+
+<p align="center"><i>If you like Kubesimplify, please ★ this repository to show your support! 🤩</i></p>
 
 This repository contains the assets required to build the Kubesimplify website and documentation. We're glad that you want to contribute! 
 
-- [Code of Conduct](code-of-conduct.md)
+## <h1>Code of conduct</h1>
```
Comment (AvineshTripathi) [inline README.md:25]: Remove this part please

Comment (KrishAryan) [reply README.md:25]: But then it will look like Kubesimplify is a part of cncf but we in reality Kubesimpilify just follows cncf code of conduct

Comment (AvineshTripathi) [reply README.md:25]: we need a code of conduct however that must be our own and not CNCFs so remove the code of conduct part for now and in future will make one(If you want to work you can take that task, sit one of the core members of kubesimplify and decide a code of conduct )

Comment (KrishAryan) [reply README.md:25]: I would love to work on that but for now I will remove code of conduct.

**Comment by KrishAryan** [comment]
@AvineshTripathi  please review this again

**Comment by AvineshTripathi** [comment]
/lgtm 
Thank you for the contribution.
Please create an issue for the code of conduct and connect to any core member to collaborate in preparing it

**Pull request merged**